Justfile

```diff
@@ -206,12 +203,9 @@ run-daemons-e2e:
 	just \
 		registry="{{registry}}" repository="{{repository}}" \
 		run-graph-builder-e2e 2>&1 &
-	PG_PID=$!
 
 	just run-policy-engine 2>&1 &
-	PE_PID=$!
 
-	trap "kill $PG_PID $PE_PID" EXIT
 	sleep infinity
```

Comment:
It feels strange to me drop drop this kill but still keep the infinite `sleep`.  Don't you want killing the `just run-daemons-e2e` call to clean up the daemons it created?